### PR TITLE
User already authenticated fix

### PR DESCRIPTION
**Pull request by MuriloFrade**

### CODE REVIEW

File: src/message/message-processor.ts [hunk 1]
```diff
@@ -46,7 +46,16 @@ export default class MessageProcessor {
         }, PARSER_ACTIONS.MESSAGE_PARSE_ERROR, message)
         continue
       }
-
+      if (message.topic === TOPIC.AUTH && message.action === AUTH_ACTIONS.REQUEST) {
+        this.services.logger.warn(
+          AUTH_ACTIONS[AUTH_ACTIONS.ALREADY_AUTHENTICATED],
+          'user is already authenticated'
```
Comment (yasserf) [inline src/message/message-processor.ts]: might be worth mentioning what user here

File: test/message/message-processorSpec.ts [hunk 2]
```diff
@@ -90,4 +90,18 @@ describe('the message processor only forwards valid, authorized messages', () =>
 
     expect(lastAuthenticatedMessage).toBe(message as any)
   })
+
+  it('does not process auth requests', () => {
```
Comment (yasserf) [inline test/message/message-processorSpec.ts]: this test doesn't make sense, processing a authentication request once should not result in an error message.

**Comment by yasserf** [comment]
@MuriloFrade what is the state of this?

**Comment by MuriloFrade** [comment]
This bug is fixed in the new client/serve release

**Pull request closed without merging**